### PR TITLE
Add instructions for module Apache Solr Search

### PR DESCRIPTION
**Pull request by MacWeber**

### CODE REVIEW

File: src/frameworks/drupal7/solr.md [hunk 1]
```diff
@@ -17,6 +29,30 @@ projects[search_api_override][version] = 1.0-rc1
...
+      
+      $environment_url = "http://" . $endpoint['host'] . ":" . $endpoint['port'] . "/solr";
+      $conf['apachesolr_default_environment'] = $environment_machine_name;
+      $conf['apachesolr_environments'][$environment_machine_name]['url'] = $environment_url;
+    }
+  }
+}
+```
+
+Replace `MACHINE_NAME_OF_SOLR_SERVER` with the Drupal machine name of the server you want to override.  The solr server must already be defined in Drupal and ideally exported to a Feature. The most comom machine name used is just `solr`
```
Comment (Crell) [inline src/frameworks/drupal7/solr.md]: Typo: "comom".

File: src/frameworks/drupal7/solr.md [hunk 2]
```diff
@@ -46,4 +82,17 @@ if (isset($_ENV['PLATFORM_RELATIONSHIPS'])) {
 
 Replace `MACHINE_NAME_OF_SOLR_SERVER` with the Drupal machine name of the server you want to override.  The solr server must already be defined in Drupal and ideally exported to a Feature.
 
-If you did not name the relationship `solr` in your `.platform.app.yaml` file, adjust the name accordingly above.  Also, if you have multiple Solr cores defined the above `foreach()` loop will not work.  Most likely you will want to name the relationships by the machine name of the Search API server they should map to and then map each one individually.
+## Relationships configuration
+If you did not name the relationship `solr` in your `.platform.app.yaml` file, adjust the name accordingly.  Also, if you have multiple Solr cores defined the above `foreach()` loop will not work.  Most likely you will want to name the relationships by the machine name of the Solr server they should map to and then map each one individually.
+
+For both modules, your `.platform.app.yaml` file must have the Solr relationship enabled, such as this snippet:
+```ini
```
Comment (Crell) [inline src/frameworks/drupal7/solr.md]: Formatting note: Newline before the code block.

File: src/frameworks/drupal7/solr.md [hunk 3]
```diff
@@ -46,4 +82,17 @@ if (isset($_ENV['PLATFORM_RELATIONSHIPS'])) {
...
+# The relationships of the application with services or other applications.
+#
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
```
Comment (Crell) [inline src/frameworks/drupal7/solr.md]: I think we can omit the inline doc here.  We usually do in inline snippets.

File: src/frameworks/drupal7/solr.md [hunk 4]
```diff
@@ -1,7 +1,19 @@
 # Using Solr with Drupal 7.x
 
 ## Requirements
+You will need either the module  [Apache Solr Search](https://www.drupal.org/project/apachesolr) **or** the module [Search API](https://www.drupal.org/project/search_api). Each have different configurations.
```
Comment (Crell) [inline src/frameworks/drupal7/solr.md]: Blank line after each header, please.

**Comment by Crell** [comment]
Thanks!  A few small comments, nothing major.

My main question, though, is if we should interleave the documentation like you're proposing (installation for module a, the module b, then configuration for a, then b) or separate them out (module a installation, module a configuration, module b installation, module b configuration.)  Thoughts?

**Comment by MacWeber** [comment]
@Crell new PR with separated pages: https://github.com/platformsh/platformsh-docs/pull/492

**Pull request closed without merging**